src/test/suite/resolveSettingsVariables.test.ts

```
@@ -0,0 +1,86 @@
+    process.env.FOO = "hello";
+    expect(resolveSettingsVariables("--vars=${env:FOO}")).toBe("--vars=hello");
+  });
+
+  it("resolves MULTIPLE ${env:VAR} placeholders in one string", () => {
+    // Regression: the previous implementation used a stateful global regex
+    // with a mutating string, so the second placeholder was skipped.
+    process.env.FOO = "alpha";
+    process.env.BAR = "beta";
+    expect(
+      resolveSettingsVariables("--vars='a=${env:FOO},b=${env:BAR}'"),
+    ).toBe("--vars='a=alpha,b=beta'");
```

ESLint

[error] Replace `⏎······resolveSettingsVariables("--vars='a=${env:FOO},b=${env:BAR}'"),⏎····).toBe("--vars='a=alpha,b=beta'"` with `resolveSettingsVariables("--vars='a=${env:FOO},b=${env:BAR}'")).toBe(⏎······"--vars='a=alpha,b=beta'",⏎····`

(prettier/prettier)